Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -344,6 +344,16 @@ class IncidentType(Enum):
 ''' HELPER FUNCTIONS '''
 
 
+def validate_supported_cortex_platform():
```

Comment:
```suggestion
def disable_for_xsiam():
```

This is not validating supported, but errors out on specific platform.